scripts/lib/api/htmlToMd.ts

```diff
@@ -301,3 +319,76 @@ function buildMathExpression(node: any, type: "math" | "inlineMath"): any {
   }
   return { type: type, value };
 }
+
+function buildMdxComponent(h: H, node: any): any {
+  const apiType = node.tagName.charAt(0).toUpperCase() + node.tagName.slice(1);
```

Comment:
Nit: consider creating a `capitalize` function in `stringUtils.tsx` since this function already has a lot going on.

Reply:
Also `componentName` might be more clear than `apiType`. You want to make clear this is referring to the specific component from MDX.